build_runner/lib/src/entrypoint/options.dart

```diff
@@ -391,23 +391,22 @@ class _TestCommand extends BuildRunnerCommand {
       if (options.outputDir == null && outputDir != null) {
         await new Directory(outputDir).delete(recursive: true);
       }
-
-      await ProcessManager.terminateStdIn();
     }
   }
 
   /// Runs tests using [precompiledPath] as the precompiled test directory.
   Future<int> _runTests(String precompiledPath) async {
     stdout.writeln('Running tests...\n');
     var extraTestArgs = argResults.rest;
-    var testProcess = await new ProcessManager().spawn(
+    var testProcess = await Process.start(
         _pubBinary,
         [
           'run',
           'test',
           '--precompiled',
           precompiledPath,
-        ]..addAll(extraTestArgs));
+        ]..addAll(extraTestArgs),
+        mode: ProcessStartMode.INHERIT_STDIO);
```

Comment:
need to update pubspec with the SDK constraint

Reply:
ya I forgot to commit that change, updated